test/cpp/jit/test_custom_class_registrations.cpp

```diff
@@ -51,9 +51,6 @@ struct Foo : torch::CustomClassHolder {
   int64_t combine(c10::intrusive_ptr<Foo> b) {
     return this->info() + b->info();
   }
-  ~Foo() {
-    // std::cout<<"Destroying object with values: "<<x<<' '<<y<<std::endl;
-  }
 };
```

Comment:
Intentional?

Reply:
Yeah. this is commented out anyways so i thought i might as well delete it.

Reply:
Hmm, ok.